MixItUp.WPF/Controls/MainControls/ChannelControl.xaml.cs

```diff
@@ -67,10 +67,14 @@ protected override Task InitializeInternal()
             this.AgeRatingComboBox.ItemsSource = EnumHelper.GetEnumNames<AgeRatingEnum>();
 
             this.StreamTitleTextBox.Text = ChannelSession.Channel.name;
-            if (ChannelSession.Channel.type != null)
+            if (ChannelSession.Channel?.type?.name != null)
```

Comment:
Wouldn't the "type?" fail if there is not Game Type assigned? Maybe add an extra check for null on the type?

Reply:
No, this syntax is a bit newer but basically translates this code:
 if (ChannelSession.Channel != null && ChannelSession.Channel.type != null && ChannelSession.Channel.type.name != null) 

So, basically if type is null then the evaluation stops and the rest of the result is null. else, it moves to check name.

Reply:
ooooh, cool :)